src/stubs.ts

```diff
@@ -144,6 +144,20 @@ export const createFigma = (config: TConfig): PluginAPI => {
       }
       return this.children.find(callback);
     }
+
+    findChild(callback) {
+      if (!this.children) {
+        return null;
+      }
+      return this.children.find(callback);
+    }
+
+    findChildren(callback) {
+      if (!this.children) {
+        return null;
+      }
+      return this.children.find(callback);
+    }
   }
```

Comment:
Is defining findChild/findChildren necessary for using `@figma-plugin/types`?

Reply:
it's coming from new version of figma.d.ts